app/views/workload-points.html

```diff
@@ -41,6 +41,9 @@
                 <tbody>
                   <tr>
```

Comment:
Could we include a check on this page that if there are errors the form is already in edit mode?

Reply:
?

Reply:
Missed this one! Will take a look 👍